.github/dependabot.yml

```diff
@@ -0,0 +1,8 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    labels:
+      - "GHA"
```

Comment:
Or probably use the `infra` tag?

Reply:
yes please ! we have 30+ labels already 😄 https://github.com/open-telemetry/opentelemetry-dotnet/labels

Reply:
Done.